Library/Formula/libechonest.rb

```diff
@@ -13,14 +11,14 @@ class Libechonest < Formula
     sha1 "1ae85cce12862896bcc2d28509854dd8e6b3d9be" => :mountain_lion
   end
 
-  depends_on 'cmake' => :build
-  depends_on 'qt'
-  depends_on 'qjson'
+  depends_on "cmake" => :build
+  depends_on "qt"
+  depends_on "qjson"
 
-  conflicts_with 'doxygen', :because => "cmake fails to configure build."
+  conflicts_with "doxygen", :because => "cmake fails to configure build."
 
   def install
     system "cmake", ".", *std_cmake_args
-    system "make install"
+    system "make", "install"
   end
```

Comment:
If you're feeling lovely: any chance a test could be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!